src/main/scala/com/fulcrumgenomics/umi/ExtractUmisFromBam.scala

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2016 Fulcrum Genomics LLC
+ * Copyright (c) $year Fulcrum Genomics
```

Comment:
I think IntelliJ did this when I refactored, weird that it did this, and weird it isn't putting in the year based on my template.